packages/@sanity/core/src/commands/dataset/copyDatasetCommand.js

```diff
@@ -7,12 +7,15 @@ import debug from '../../debug'
 
 const helpText = `
 Options
-  --wait Waits for the operation to finish
+  --background Start the copy and return without waiting for it to finish
```

Comment:
Wouldn't a better wording be: _Start the copy without waiting for it to finish_

Reply:
Sure! 🙂 Fixed in 7841ced59